exec/java-exec/src/main/java/org/apache/drill/exec/work/foreman/Foreman.java

```diff
@@ -417,6 +417,9 @@ private void parseAndRunPhysicalPlan(final String json) throws ExecutionSetupExc
 
   private void runPhysicalPlan(final PhysicalPlan plan) throws ExecutionSetupException {
     validatePlan(plan);
+    //Marking endTime of Planning
+    queryManager.markPlanningEndTime();
```

Comment:
Nit. The next line is also part of planning, though should execute quite quickly.